Week4/app/src/main/java/com/isabelledionisius/mymovieapp/Movie.kt

```diff
@@ -0,0 +1,59 @@
+package com.isabelledionisius.mymovieapp
+
+import android.os.Parcel
+import android.os.Parcelable
+
+ class Movie(
+    val id: Int,
+    val releaseDate: String,
+    val title: String,
+    val summary: String,
+    val genre: String,
+    val poster: Int
+) : Parcelable {
```

Comment:
You don't have to generate parcels yourself.

Just add `@Parcelize` to the class header, and that's it!